vicon_mujoco_init/robot_init.py

```diff
@@ -0,0 +1,155 @@
+import math
+import time
+import context
+import o80
+import pam_mujoco
+import vicon_transformer
+import numpy as np
+from scipy.spatial.transform import Rotation as R
+
+# to run this demo, start pam_mujoco with mujoco_id "vicon_robot_init"
+# pam_mujoco vicon_robot_init && python3 <pathToCode>/pam_demos/vicon_mujoco_init/robot_init.py
+
+# read a test vicon frame from file
+print('read a test vicon frame from file')
```

Comment:
Executable Python code should always be put in an `if __name__ == "__main__":`-block to prevent unintended execution of the code when the script is imported somewhere (there are some tools, e.g. Sphinx, which import files to process them).

To avoid global variables, I would even go a step further and suggest the following pattern:
```
def main():
    # code here

if __name__ == "__main__":
    main()
```